### DISCUSSION TITLE
query without updating ids

### DISCUSSION THREAD

**Discussion by FreddieGilbraith** [GothenburgBitFactory/taskwarrior#2546] [Q&A] [answered]
Hi,

I'd like to integrate my current active task into my shell prompt, using `task +ACTIVE export | jq 'some script here'`, but as that command is run on every input to my shell, it's updating the ids of my tasks while I interact with them.

Is there a way to run `export` (or any command) and force the ids not to update?

Thanks

**Comment by tbabej** [comment, marked as answer]
Indeed! You can disable the `gc` mechanism by setting `gc=0` configuration variable, ideally using an override on the command itself:

`task rc.gc=0 +ACTIVE export | jq ...`

**Reply by FreddieGilbraith** [reply]
Ahhh, I THOUGH I remembered something like that! Brilliant, thanks very much :)
